edunext_openedx_extensions/ednx_microsites/tests/test_change_domain.py

```diff
@@ -0,0 +1,33 @@
+"""This module include a class that checks the command change_domain.py"""
+
+from io import StringIO
+
+from django.test import TestCase
+from django.core.management import call_command
+
+from edunext_openedx_extensions.ednx_microsites.models import Microsite
+
+
+class ChangeDomainTestCase(TestCase):
+    """ This class checks the command change_domain.py"""
+
+    # pylint: disable=no-member
```

Comment:
This is not needed, we can use the `Microsite.object.create` when needed, and avoid this definition